gnat/lsp_server.gpr

```diff
@@ -47,7 +47,7 @@ project LSP_Server is
    package Compiler is
       for Default_Switches ("Ada") use LSP.Compiler'Default_Switches ("Ada");
       for Switches ("lsp-ada_driver.adb") use
-        LSP.Ada_Switches & ("-gnateDVERSION=""" & VERSION & """");
+        LSP.Ada_Switches & ("-gnatX", "-gnateDVERSION=""" & VERSION & """");
```

Comment:
Let's replace `LSP.Ada_Switches` with `LSP.Compiler'Default_Switches ("Ada")` instead?

Reply:
yes it's needed to have debug info for lsp-ada_driver.adb

Reply:
I agree.